frontend/src/pages/BYONImages/BYONImageAccelerators.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Spinner,
+  LabelGroup,
+  Label,
+  StackItem,
+  Stack,
+  Tooltip,
+  Button,
+} from '@patternfly/react-core';
+import { PlusIcon } from '@patternfly/react-icons';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { BYONImage } from '~/types';
+import { AcceleratorKind } from '~/k8sTypes';
+import { FetchState } from '~/utilities/useFetchState';
+
+type BYONImageAcceleratorsProps = {
+  image: BYONImage;
+  acceleratorProfiles: FetchState<AcceleratorKind[]>;
+};
+
+export const BYONImageAccelerators: React.FC<BYONImageAcceleratorsProps> = ({
+  image,
+  acceleratorProfiles,
+}) => {
+  const navigate = useNavigate();
+
+  const [data, loaded, loadError] = acceleratorProfiles;
+
+  const recommendedAcceleratorProfiles = data
+    .filter((accelerator) =>
+      image.recommendedAcceleratorIdentifiers?.includes(accelerator.spec.identifier),
+    )
+    .map((accelerator) => accelerator.spec.displayName);
+
+  if (loadError) {
+    return <>{'-'}</>;
+  }
+
+  if (!loaded) {
+    return <Spinner size="sm" />;
+  }
+
+  return (
+    <Stack>
+      {recommendedAcceleratorProfiles.length > 0 && (
+        <StackItem>
+          <LabelGroup isCompact>
+            {recommendedAcceleratorProfiles.map((acceleratorName, i) => (
+              <Label key={acceleratorName + i} color="blue" variant="filled" isTruncated isCompact>
+                {acceleratorName}
+              </Label>
+            ))}
+          </LabelGroup>
+        </StackItem>
+      )}
+      <StackItem>
+        {image.recommendedAcceleratorIdentifiers?.length > 0 ? (
+          <Tooltip
+            content={`This image is compatible with accelerators with the identifier ${image.recommendedAcceleratorIdentifiers.join(
+              ', ',
+            )}.`}
+          >
+            <Label
+              color="blue"
+              variant="outline"
+              onClick={() => {
+                if (image.recommendedAcceleratorIdentifiers?.length) {
+                  navigate('/acceleratorProfiles/create/', {
+                    state: {
+                      acceleratorIdentifiers: image.recommendedAcceleratorIdentifiers,
+                    },
+                  });
+                }
+              }}
```

Comment:
Clicking on this link causes the page to first navigate and then does a full page refresh.

Also on that page is I press `cancel` I'm brought to the `Accelerator profiles` page and not back to the `Notebook image settings` page.

Reply:
> Also on that page is I press cancel I'm brought to the Accelerator profiles page and not back to the Notebook image settings page.

This was intended by UX

> Clicking on this link causes the page to first navigate and then does a full page refresh.

I am using href now and not navigate

Reply:
You need to follow the react-router example for using `Link` here otherwise the page still refreshes: http://v4-archive.patternfly.org/v4/components/label#router-link